gdx/src/com/badlogic/gdx/scenes/scene2d/ui/Table.java

```diff
@@ -146,6 +151,19 @@ public void setBackground (Drawable background) {
 		else if (padTopOld != padTopNew || padLeftOld != padLeftNew || padBottomOld != padBottomNew || padRightOld != padRightNew)
 			invalidate();
 	}
+	
+	/** @param background May be null to clear the background. */
+	public void setBackground(Color color){
+		TextureRegionDrawable d = null;
+		if(color!=null){
+			Pixmap p = new Pixmap(1, 1, Format.RGBA8888);
+			p.setColor(color);
+			p.fill();
+			d = new TextureRegionDrawable(new TextureRegion(new Texture(p)));

```

Comment:
Yeah that's definitely a problem. I could return it to leave that responsibility to the programmer but I'm not sure if I like this option.